lib/debug.nix

```diff
@@ -77,7 +77,7 @@ rec {
                (modify depth snip x)) y;
 
   /* A combination of `traceVal` and `traceSeq` */
-  traceValSeqFn = f: v: traceVal f (builtins.deepSeq v v);
+  traceValSeqFn = f: v: traceValFn f (builtins.deepSeq v v);
```

Comment:
Huh, I thought we merged that line already?

Reply:
it's not up to date. I've started modifying the code to adopt somewhat the module backend so it might be best not to review it now